app/services/umrdr/deposits_okay_service.rb

```diff
@@ -0,0 +1,59 @@
+module Umrdr
+
+  require 'rubygems'
+  require 'net/ldap'
+
+  ENV['LDAPTLS_CACERT'] = '../../../config/incommon_server_ca_cert.crt';
```

Comment:
Is this required for the ruby code to run in production or staging?  I think this root cert is already installed in our infrastructure.  This could probably be a comment for any devs that want to test locally.